content/docs/reference/state.md

```diff
@@ -1,140 +1,212 @@
 ---
 title: State and Backends
+description:
+keywords:
 expanded_url: /docs/reference/concepts/
 menu:
   reference:
     parent: concepts
     weight: 7
 ---
 
+## State
+
 Pulumi stores its own copy of the current state of your infrastructure. This is often simply called _state_, and is
-stored in transactional snapshots we call _checkpoints_. A checkpoint is recorded by Pulumi at various points so that
-it can operate reliably, whether that means diffing goal state versus current state during an update, recovering from
-failure, or destroying resources accurately to clean up afterwards, for example. Because state is critical to how Pulumi
+stored in transactional snapshots we call _checkpoints_. A _checkpoint_ is recorded by Pulumi at various points so that
+it can operate reliably &mdash; whether that means diffing goal state versus current state during an update, recovering from
+failure, or destroying resources accurately to clean up afterwards. Because _state_ is critical to how Pulumi
 operates, this page gives an overview of the options.
 
 ## Backends
 
-Pulumi supports multiple so-called _backends_ for storing this state. There are two primary kinds:
-
-1. **Web**: a backend accessed by the CLI through REST API calls
-2. **Filesystem**: a backend stored locally on your computer's filesystem
-
-By default, the CLI uses a web backend hosted at https://app.pulumi.com. This is a free service offered by Pulumi
-that also offers advanced tiers for team and enterprise features. It is possible to opt-in to the filesystem backend.
-
-We think that using the web backend and the CLI together provides the right combination of usability, safety,
-and security for most users. But the filesystem backend provides more flexibility for those who need it.
-
-### Web Backend Features
-
-The web backend stores all checkpoint state securely, and the CLI always communicates directly with your cloud provider
-using client-side authentication. All state is encrypted in transit and at rest.
-
-Although backends are primarily responsible for storing state reliably, the web backend also supports additional
-features that the filesystem backend does not. In particular, the web backend supports keeping track of full
+Pulumi supports multiple so-called _backends_ for storing your infrastructure state. By default, the CLI uses a web backend hosted at [app.pulumi.com](https://app.pulumi.com). Pulumi offers this backend as a free service with [advanced tiers](https://www.pulumi.com/pricing/) for team and enterprise features. Using the `pulumi.com` backend and the CLI together provides the right combination of usability, safety,
+and security for most users. But you can opt into the filesystem or cloud storage backend for more flexibility if you need it.
```

Comment:
1. "providers a great combination of"
2. "However, if you would prefer to manage this yourself, you can do so by opting into the filesystem or cloud storage backend for more flexibility if you need it"

Reply:
Thank you - applied.